package.json

```diff
@@ -27,6 +27,7 @@
     "prop-types": ">=15.7.2",
     "react": "^17.0.2",
     "react-big-calendar": "^0.38.1",
+    "react-brackets": "^0.4.7",
```

Comment:
We usually require new dependencies to be approved before adding them to the project.

This particular dependency is not very healthy, and I do not think we should add it for the following reasons:
1. It is a pre-1.0 package, meaning it is open to breaking changes and has no expectation of stability.
2. It is not popular and has a limited community, meaning there are not many eyes on it to detect bugs and security issues, and not much incentive for the maintainer or the users to continue developing and supporting it.
3. The maintenance is sparse and inconsistent, meaning there isn't someone actively devoted to keeping it secure and up to date with the rest of the eco-system.

That is a recipe for bugs, security issues, and difficulties when upgrading other packages. It's package health score is only 56/100.

Reply:
I see, we use it for our visual display of brackets. Perhaps we could argue that this can be a temporary implementation and have our own component to be developed in the near future?

Reply:
Yeah, I see that y'all are using it quite extensively.

I don't think it's reasonable at this stage to ask you to do a full rewrite of the bracket UI, so your suggestion seems like the best path forward.

Reply:
https://github.com/gordon-cs/RecIM-Docs/issues/203